shell/browser/api/electron_api_web_contents.cc

```diff
@@ -3708,6 +3710,33 @@ void WebContents::UpdateHtmlApiFullscreen(bool fullscreen) {
   }
 }
 
+std::string WebContents::GetMediaSourceID(
```

Comment:
Can you also move the implementation code to below `SetWebRTCIPHandlingPolicy`? Making the implementation and declaration have same order will make the code easier to maintain.